### PR TITLE
fix: streamReadableWebToString encoding

### PR DESCRIPTION
**Pull request by nitedani**

### CODE REVIEW

File: vike/node/runtime/html/stream.ts [hunk 1]
```diff
@@ -115,15 +115,17 @@ async function streamReadableNodeToString(readableNode: StreamReadableNode): Pro
...
     const { done, value } = await reader.read()
-    if (done) {
-      break
-    }
-    str += value
+    if (done) break
+    str += decoder.decode(value, { stream: true })
   }
+
+  str += decoder.decode()
```
Comment (brillout) [inline vike/node/runtime/html/stream.ts:128]: Curious: why this line?

Comment (nitedani) [reply vike/node/runtime/html/stream.ts:128]: image

Comment (nitedani) [reply vike/node/runtime/html/stream.ts:128]: The final decoder.decode() is actually only necessary in a very specific scenario:
If the very last chunk of the stream ends with the beginning of a multi-byte character, that incomplete character data will be held by the decoder and not output. The final decoder.decode() call will then output a replacement character (usually �) for this incomplete sequence.

**Comment by brillout** [comment]
Released in `0.4.184`.

**Pull request merged**